scaling_performance/optimizing_storage.adoc

```diff
@@ -13,6 +13,81 @@ toc::[]
+
+WARNING: ^(3)^ Vendor GlusterFS, Vendor S3, and Vendor Swift supportability and configurability may vary.
+
+=== Storage Backend Recommendations
+Below is a matrix that summarizes which storage backend types are Recommended, Supported, and Configurable for the given
+{product-title} application and useful attributes.
+[format="csv",width="70%",cols="10"]
+[options="header]
+|=======================
+Type,Tied to Node?,ROM,RWX,Registry,Scaled Registry,etcd,Metrics,Logging,Apps
+hostPath,Yes,No,No,Not Configurable,Not Configurable,Recommended,Recommended,Recommended,Configurable
+Block,No^(1)^,Yes^(2)^,No,Supported,Configurable,Recommended,Recommended,Recommended,Recommended
+File,No,Yes^(2)^,Yes,Supported,Configurable,Configurable,Configurable,Configurable,Recommended
+Object,No,Yes,Yes,Recommended,Recommended,Not Configurable,Not Configurable,Not Configurable,Recommended^(3)^
+|=======================
+NOTE: ^(1)^ This does not apply to HDD, SSD, NVMe, Raw Physical Disk, VM Physical Disk, VMDK, and LVM.  These storage devices are tied to the nodes.
+
+NOTE: ^(2)^ This does not apply to Physical Disk, VM Physical Disk, VMDK, Loopback over NFS, AWS EBS, and Azure Disk.
+
+NOTE: ^(3)^Object Storage is not used via {product-title}'s PVs.  Apps must integrate with the Object Storage API.
+
+==== Specific Application & Storage Recommendations
+===== Registry
+In a non-scaled/HA {product-title} Internal Docker Registry cluster deployment, the preferred Storage Backend is Ojbect Storage
+followed by Block Storage.  The Storage Backend does not need to support ReadWriteMany Access Mode.  The Storage Backend *must
+ensure Read-After-Write-Consistency*.  All Network Attached File Storage Backends are expressly *not recommended* for Internal
+Docker Registry cluster deployment.
+
+===== Scaled Registry
+In a scaled/HA {product-title} Internal Docker Registry cluster deployment, the preferred Storage Backend is Ojbect Storage.
+The Storage Backend *must support ReadWriteMany* Access Mode and *must ensure Read-After-Write-Consistency*.  Block Storage is
+expressly *not recommended* for a scaled/HA {product-title} Internal Docker Registry cluster deployment.  All Network File
+Storage Backends are expressly *not recommended* for Internal Docker Registry cluster deployment.
+
+===== etcd
+For etcd, the preferred Storage Backend is Block Storage.  For the best etcd reliability, the lowest consistent latency Storage Backend is preferrable.
+
+===== Metrics
+In {product-title} Hosted Metrics cluster deployment, the preferred Storage Backend is Block Storage.  All Network Attached File
+Storage Backends are expressly *not recommended* for Hosted Metrics cluster deployment.
+
+===== Logging
+In {product-title} Hosted Logging cluster deployment, the preferred Storage Backend is Block Storage.  All Network Attached File
+Storage Backends are expressly *not recommended* for Hosted Logging cluster deployment.
```

Comment:
I'll add a note later in the text unless you think it should added somewhere here.